src/r2b2/contest.py

```diff
@@ -7,7 +7,8 @@
 class ContestType(Enum):
     """Enum indicating what type of vote variation was used in the contest.
 
-    TODO: Add additional vote variations from VVSG.
+    Todo:
+        Add additional vote variations from VVSG.
```

Comment:
I only changed it because I the `Todo: `  is an accepted keyword for Sphinx with our style of Docstrings, while `TODO: ` is not and simply appears as plain text in the Documentation. Using this format, it will not be generated in the Documentation on RTD, even though it appears in the Docstring. 

We could switch to `# TODO: ` comments below the Docstring entirely.

Reply:
Hmm - puzzling. When I look at r2b2/dist/docs/autoapi/r2b2/contest/index.html I see nothing in the rendered documentation relating to Todo:, though I see it via the [source] link. But with TODO: as before I see it as text in the documentation.

Reply:
Exactly, the `Todo: ` will be ignored as it is not useful documentation for the user, but a note for developers.

Reply:
Ahh - I misunderstood. Thanks. Good to know we have both options.
I'm guessing that this particular Todo is more of a research agenda for now, working with the SHANGRLA framework. Perhaps noting that as an issue would be more helpful.